tests/support/test.tcl

```diff
@@ -258,7 +258,7 @@ proc test {name code {okpattern undefined} {tags {}}} {
     }
 
     if {$::traceleaks} {
```

Comment:
Traceleaks are no longer useful. Can we consider removing this directly?

Reply:
there may be other dead code in tcl. I think it is irrelevant, so let’s keep it for now.